impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -368,19 +378,31 @@ class AllocatedTextureSourceVK final : public TextureSourceVK {
     return resource_->image_view.get();
   }
 
+  vk::ImageView GetRenderTargetView() const override {
+    return resource_->rt_image_view.get();
+  }
+
+  bool IsOnscreen() const override { return false; }
+
  private:
   struct ImageResource {
     UniqueImageVMA image;
     vk::UniqueImageView image_view;
+    vk::UniqueImageView rt_image_view;
 
     ImageResource() = default;
 
-    ImageResource(ImageVMA p_image, vk::UniqueImageView p_image_view)
-        : image(p_image), image_view(std::move(p_image_view)) {}
+    ImageResource(ImageVMA p_image,
+                  vk::UniqueImageView p_image_view,
+                  vk::UniqueImageView p_rt_image_view)
+        : image(p_image),
+          image_view(std::move(p_image_view)),
+          rt_image_view(std::move(p_rt_image_view)) {}
 
     ImageResource(ImageResource&& o) {
       std::swap(image, o.image);
       std::swap(image_view, o.image_view);
+      std::swap(rt_image_view, o.rt_image_view);
```

Comment:
Can't this just be `ImageResource(ImageResource&&) = default`?

Reply:
Done